.gitignore

```diff
@@ -34,3 +34,17 @@ Commons/core/src/main/i18n/translations/*
 
 # Minecraft generated files
 maps.zip
+banned-ips.json
+banned-players.json
+ops.json
+help.yml
+server.properties
+sportpaper.yml
+update
+usercache.json
+whitelist.json
+logs/
+plugins/
+world/
+world_nether/
+world_the_end/
```

Comment:
Aren't these inside the `matches/` directory?

Reply:
For matches, maybe (it’s been a while) - but﻿ I believe these are the directories that are generated when the server starts initially